hidden-posts.php

```diff
@@ -253,13 +253,15 @@ public function render_metabox( WP_Post $post ) {
 
 }
 
-new Hidden_Posts();
+$hidden_posts = new Hidden_Posts();
+$hidden_posts->run();
 
 /**
  * Helper function to get hidden posts.
  *
  * @return array Array of Post IDs.
  */
 function vip_get_hidden_posts() { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedFunctionFound
-	return Hidden_Posts::get_posts();
+	$hidden_posts = new Hidden_Posts();
```

Comment:
How does instantiating a class, and calling a method on it, benefit over the static method call that it had before?

How does this change, tie in with the move to simplify the constructor?

Reply:
Thanks for your review, @GaryJones. 

> How does instantiating a class, and calling a method on it, benefit over the static method call that it had before?

It does not and even made is worse as it increased the lines of code for no reason. 🤭

> How does this change, tie in with the move to simplify the constructor?

When creating this PR, I was thinking I would have to change this particular part as well. Now, that I take another look, I see that the static function is independent of the rest of the code, and its only purpose is to return an array with the IDs of all hidden posts. This function does not rely on any of the hooks and therefore does not require an instance. As a result of that, I reverted 

```
$hidden_posts = new Hidden_Posts();
return $hidden_posts->get_posts();
```

to 

```
return Hidden_Posts::get_posts();
```

Would you mind doing another review?